recipe/meta.yaml

```diff
@@ -12,79 +12,32 @@ source:
   patches:
     - clang.patch  # [osx]
     - prepend-dll.patch  # [win]
-    # Fixes compilation problems with JPEG.
-    # https://lists.osgeo.org/pipermail/gdal-dev/2016-January/043443.html
-    - disable_jpeg12.patch  # [win]
-    - windowshdf5.patch  # [win]
-    - multiprocessor.patch  # [win]
     - install_scripts.patch
 
 build:
-  number: 2
-  skip: True  # [py36 and linux]
+  number: 3
```

Comment:
I bumped the build number here but I am also planning to move the previous build number to a "broken" channel to avoid confounding the conda solver.